packages/docs/docs/getting-started/media.mdx

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 13
+slug: /media
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Media
+
+In this chapter we will take a look at all the different types of media Motion
+Canvas lets you add to your animations.
+
+## Audio
+
+One type of media Motion Canvas supports is audio. To add an audio track to your
+animation you would need to edit your `project.ts` file. First we need to import
```

Comment:
nit: "would" is in a little bit of weird tense here. "will," or omitting the word entirely is a bit more straightforward:
> To add an audio track to your animation, you need to edit your `project.ts` file.